root/etc/e-smith/events/actions/nethserver-tancredi-conf

```diff
@@ -58,3 +58,6 @@ if [[ ! -f ${dst_file} ]]; then
     echo 'adminpw = "'$(head /dev/urandom | tr -dc a-z0-9 | head -c 10)'"' >> ${dst_file}
     echo 'userpw = "'$(head /dev/urandom | tr -dc a-z | head -c 6)'"' >> ${dst_file}
 fi
+
+# Launch upgrades
+/usr/bin/scl enable rh-php56 -- php /usr/share/tancredi/src/Upgrade.php
```

Comment:
This is my wish: the script out of the src/ namespace root, used by the public HTTP endpoints

Can it work?

```suggestion
/usr/bin/scl enable rh-php56 -- php /usr/share/tancredi/upgrade.php
```